http/file_server.ts

```diff
@@ -515,10 +524,11 @@ function dirViewerTemplate(dirname: string, entries: EntryInfo[]): string {
         } else {
           link = "../".repeat(depth);
         }
-        return `<a href="${link}">${escape(path)}</a>`;
+        return html`
+          <a href="${link}">${escape(path)}</a>
+        `;
```

Comment:
Maybe better to not use `html` here to avoid some rendering changes pointed by @lionel-rowe ?